Packs/Sigma/Playbooks/playbook-Sigma_-_Query_using_Sigma_rules.yml

```diff
@@ -0,0 +1,1344 @@
+id: Sigma - Query using Sigma rules
+version: -1
+name: Sigma - Query using Sigma rules
```

Comment:
I think the prefix is unnecessary, wdyt?

Reply:
Isn't it required as it is a unique pack? 
I thought that down the line we might have more than one PB and it will be easier to find it in the list.